themes/red-theme/lms/templates/footer.html

```diff
@@ -0,0 +1,111 @@
+## mako
+<%namespace name='static' file='static_content.html'/>
+<%!
+from django.core.urlresolvers import reverse
+from django.utils.translation import ugettext as _
+from microsite_configuration.templatetags.microsite import platform_name
+%>
+
+<div class="wrapper wrapper-footer">
+  <footer>
+    <!-- This is super-ugly, don't use it! -->
```

Comment:
NIT: this is an HTML comment where all other informative comments are formatted in python ``##``. Mind syncing it up or removing it if its not needed?